### ISSUE TITLE
Problems using RetinaFace

### ISSUE THREAD

**Issue by ItamarSafriel** [timesler/facenet-pytorch#154]
Hello, 
In order to get less false positive face detection I tried switching MTCNN with RetinaFace using [this](https://github.com/biubug6/Pytorch_Retinaface) open source implementation. After extracting all faces from a given image I am passing them to FaceNet but I think I need to post-process my face images. I aligned and resized them but I am still getting strange results. Using a toy datasets (with 7 identities) and the [Umap algorithm](https://umap-learn.readthedocs.io/en/latest/) I got the following:

1. **Reduced Embeddings Using MTCNN**:
![facenet_output](https://user-images.githubusercontent.com/49122015/109830831-86249a00-7c47-11eb-94cc-e56cdc0934b6.png)

2. **Reduced Embeddings Using RetinaFace**:
![retinaface_output](https://user-images.githubusercontent.com/49122015/109830855-8de43e80-7c47-11eb-8c30-265c195e392e.png)

As you can see, in the first plot the embeddings of different identities are well separated to clusters but 
when using RetinaFace this is not the case. 

DId any one run into this problem? Any ideas? I already looked through the code for post processing steps (such as ```fixed_image_standardization```) but nothing I tried worked. 

Thanks in advance!

**Comment by shashi-banger** [comment]
This might help.
```
from PIL import Image
import numpy as np

f = Image.open('input_file')
f = np.array(f)

face_tensor = torch.tensor((f-128.0)/128.0, dtype=torch.float).permute(2, 0, 1).cuda()

embedding = resnet(face_tensor)
```

Normalizing retinaface output to (-1.0,1.0) range helped me, when I was facing similar issue that you have reported.

**Comment by sellaziz** [comment]
@ItamarSafriel 
Hi, did you manage to fix your issue ? I have a similar issue

**Comment by Sinuhe96** [comment]
I thought MTCNN is SOTA in face detection? I would increase the min_face_size and use higher thresholds to decrease false detection. Accept only faces with high confidence would also help.
